hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmSnapshot.java

```diff
@@ -215,6 +225,12 @@ private  OzoneFileStatus denormalizeOzoneFileStatus(
         omKeyInfo, fileStatus.getBlockSize(), fileStatus.isDirectory());
   }
 
+  private KeyInfoWithVolumeContext denormalizeKeyInfoWithVolumeContext(
+      KeyInfoWithVolumeContext k) {
+    return new KeyInfoWithVolumeContext(k.getVolumeArgs().orElse(null),
```

Comment:
Would using its `Builder` be better than calling the constructor directly?

Reply:
done